Index: sympy/doc/src/conf.py
===================================================================
--- sympy.orig/doc/src/conf.py
+++ sympy/doc/src/conf.py
@@ -30,7 +30,7 @@ sys.path = ['ext'] + sys.path
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.addons.*') or your custom ones.
 extensions = ['sphinx.ext.autodoc', 'sphinx.ext.linkcode',
-              'sphinx_math_dollar', 'sphinx.ext.mathjax', 'numpydoc',
+              'sphinx_math_dollar', 'sphinx.ext.imgmath', 'numpydoc',
               'sphinx_reredirects', 'sphinx_copybutton',
               'sphinx.ext.graphviz', 'sphinxcontrib.jquery',
               'matplotlib.sphinxext.plot_directive', 'myst_parser',
