Index: sympy/doc/src/conf.py
===================================================================
--- sympy.orig/doc/src/conf.py
+++ sympy/doc/src/conf.py
@@ -43,7 +43,7 @@ autodoc_inherit_docstrings = False
 # MathJax file, which is free to use.  See https://www.mathjax.org/#gettingstarted
 # As explained in the link using latest.js will get the latest version even
 # though it says 2.7.5.
-mathjax_path = 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-AMS_HTML-full'
+mathjax_path = 'mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML'
 
 # See https://www.sympy.org/sphinx-math-dollar/
 mathjax_config = {
